src/components/FilterPanel.tsx

```diff
@@ -44,6 +45,7 @@ export default function FilterPanel({
   const [grades, setGrades] = useState(params.grades);
   const [majors, setMajors] = useState(params.majors);
   const [states, setStates] = useState(params.states);
+  const [schools, setSchools] = useState(params.schools);
```

Comment:
If we do this we could probably get rid of the `|| []` elsewhere.
```suggestion
  const [grades, setGrades] = useState(params.grades || []);
  const [majors, setMajors] = useState(params.majors || []);
  const [states, setStates] = useState(params.states || []);
  const [schools, setSchools] = useState(params.schools || []);
```

Reply:
problem is that it complains that they are `undefined`. I checked in the console and they go from being an empty array to undefined. 

I could do this change inside the component itself.